packages/next/src/build/webpack-build/impl.ts

```diff
@@ -333,7 +333,9 @@ export async function webpackBuildImpl(
       err.code = 'INVALID_RESOLVE_ALIAS'
       throw err
     }
-    const err = new Error('Build failed because of webpack errors') as NextError
+    const err = new Error(
+      `Build failed because of ${process.env.NEXT_RSPACK ? 'rspack' : 'webpack'} errors`
```

Comment:
I noticed the [JSON map](https://github.com/vercel/next.js/pull/76682/files#diff-58a77e62796608e78f8ef4373ddafad4dc329d2fb96df8aa0b820318cea68946R662) was added, and was wondering if there was any standard way to re-use the error message from the map instead of having to hardcode it here?

My gut tells me these could potentially get out of sync or not match up exactly, but maybe it's not a concern.

Reply:
The mapping file is generated statically from the code -- it's never manually edited, so there's no risk of going out of sync.

Reply:
Ah that makes a lot more sense haha
Thank you for the context.